src/components/PokemonList.js

```diff
@@ -1,23 +1,26 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
 
 export default function PokemonList() {
   const [pokemon, setPokemon] = useState([]);
-
-  async function loadPokemon() {
-    try {
-      const response = await fetch("https://pokeapi.co/api/v2/pokemon");
-      const data = await response.json();
-      setPokemon(data.results);
-    } catch (error) {
-      console.log(error);
+  useEffect(() => {
+  
+    async function loadPokemon() {
+      try {
+        const response = await fetch("https://pokeapi.co/api/v2/pokemon");
+        const data = await response.json();
+        setPokemon(data.results);
+      } catch (error) {
+        console.log(error);
+      }
     }
-  }
+    loadPokemon()
+},[])
 
   return (
     <main>
-      <button type="button" onClick={loadPokemon}>
+      {/* <button type="button" onClick={loadPokemon}>
```

Comment:
oh so you disabled your button because you werent able to accesss the url in the data? I used another state to save that value, if you wanna check: https://github.com/kayachakmak/challenges/pull/102/files